source/php/App.php

```diff
@@ -53,4 +57,13 @@ public function enqueueScripts()
 
         wp_enqueue_script('api-volunteer-manager-js');
     }
+
+    /**
+     * Update ACF settings
+     * @return void
+     */
+    public function updateAcfSettings()
+    {
+        acf_update_setting('google_api_key', defined('GOOGLE_API_KEY') ? GOOGLE_API_KEY : '');
```

Comment:
This will trigger a db update on each page load. Use filter instead.

Reply:
good catch!